src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -285,6 +294,31 @@ export default class HaAutomationConditionRow extends LitElement {
     }
   }
 
+  private async _renameCondition(): Promise<void> {
+    const describedCondition = describeCondition(this.condition);
+    const alias = await showPromptDialog(this, {
```

Comment:
Can we fix the prompt dialog? The input element should be a lot wider. It’s too small.

Reply:
Fixed the prompt dialog, and made it a little wider.